Dockerfile

```diff
@@ -197,6 +197,10 @@ RUN /builder-scripts/download-repo https://github.com/nextstrain/auspice release
 # Add evofr for forecasting
 RUN pip3 install evofr
 
+# Add NCBI Datasets command line tools for access to NCBI Datsets Virus Data Packages
+RUN curl -fsSL -o /final/bin/datasets https://ftp.ncbi.nlm.nih.gov/pub/datasets/command-line/v2/linux-amd64/datasets
+RUN curl -fsSL -o /final/bin/dataformat https://ftp.ncbi.nlm.nih.gov/pub/datasets/command-line/v2/linux-amd64/dataformat
```

Comment:
Let's prefix both of these with `ncbi-`, e.g. to get `ncbi-datasets`. An unscoped `datasets` command seems bound to be confusing.

Reply:
Ah yup, I can see how an unscoped `datasets` command can be confusing within the Nextstrain context. 
Is it also possible to also prefix with `ncbi-` in conda-base?

Reply:
Or do you mean to do something similar to Nextclade 2 like so: 

```Suggestion
# Add NCBI Datasets command line tools for access to NCBI Datsets Virus Data Packages
RUN curl -fsSL -o /final/bin/ncbi-datasets https://ftp.ncbi.nlm.nih.gov/pub/datasets/command-line/v2/linux-amd64/datasets \
 && ln -sv datasets /final/bin/ncbi-datasets
RUN curl -fsSL -o /final/bin/ncbi-dataformat https://ftp.ncbi.nlm.nih.gov/pub/datasets/command-line/v2/linux-amd64/dataformat \
 && ln -sv dataformat /final/bin/ncbi-dataformat
```

Reply:
Ah, I meant just supporting e.g. `ncbi-datasets`, not `datasets` too.

But the question about Conda is a good one. Hmm. I assumed we were going to have to package them ourselves and thus could make that choice, but I see now that there is an `ncbi-datasets-cli` package. Despite the package name being qualified, it installs the unqualified `datasets` and `dataformat` commands. (I find it ridiculous that NCBI thinks it's ok to use such generic names!)

I think we either:

- Accept `datasets` and `dataformat` for all runtimes
- Repackage NCBI Datasets' CLI ourselves for Conda with the qualified binary names

Reply:
I lean towards keeping `datasets` and `dataformat` as is so that our uses of them follow the NCBI docs and also less for us to maintain 🤷

Reply:
Yeah, agreed, though I'll cringe when we run into our first issue with the generic names… :see_no_evil:

Reply:
I guess we'll see 🤞